integration_tests/pages/alertsPage.ts

```diff
@@ -8,4 +8,28 @@ export default class AlertsPage extends Page {
   alertsList = (): PageElement => cy.get('.hmpps-alert-card-list')
 
   alertsEmptyState = (): PageElement => cy.get('[data-qa=alerts-empty-state]')
+
+  paginationHeader = (): PageElement => cy.get('.hmpps-paged-list-header .moj-pagination')
+
+  paginationSummaryHeader = (): PageElement => cy.get('.hmpps-paged-list-header .moj-pagination__results')
+
+  paginationFooter = (): PageElement => cy.get('.hmpps-paged-list-footer .moj-pagination')
+
+  paginationSummaryFooter = (): PageElement => cy.get('.hmpps-paged-list-footer .moj-pagination__results')
+
+  paginationPreviousLink = (): PageElement => cy.get('.moj-pagination__item.moj-pagination__item--previous')
+
+  paginationNextLink = (): PageElement => cy.get('.moj-pagination__item.moj-pagination__item--next')
+
+  paginationCurrentPage = (): PageElement => cy.get('.moj-pagination__item.moj-pagination__item--active')
+
+  paginationHeaderPageLink = (): PageElement =>
+    cy.get(
+      '.hmpps-paged-list-header .moj-pagination__item:not(.moj-pagination__item--active):not(.moj-pagination__item--previous):not(.moj-pagination__item--next)',
+    )
+
+  paginationFooterPageLink = (): PageElement =>
+    cy.get(
+      '.hmpps-paged-list-footer .moj-pagination__item:not(.moj-pagination__item--active):not(.moj-pagination__item--previous):not(.moj-pagination__item--next)',
+    )
```

Comment:
I wonder if this could be cleaned up to be its own object so it can then be used across multiple - so rather than separate methods its something like:

```typescript
pagination = () => {
  header: () => {}
  summaryHeader: () => {}
  // etc 
}
```

then we can move the object to another file so it ends up being `pagination = () => paginationObject`

Reply:
Good idea... will implement with next PR